node/network/availability-distribution/src/requester/fetch_task/mod.rs

```diff
@@ -308,10 +313,10 @@ impl RunningTask {
 			// Ok, let's store it and be happy:
 			self.store_chunk(chunk).await;
 			succeeded = true;
-			_span.add_string_tag("success", "true");
+			span.add_string_tag("success", "true");
```

Comment:
I dont think this is needed.

Reply:
In this case it differentiates between whether the storage operation is included, or not in the span, which influences the duration of the span.